src/red-lentil-soup.md

```diff
@@ -0,0 +1,29 @@
+# Red Lentil Soup
+
+- 🍳Cook time: 10 min
+
+## Ingredients
+
+- Red Lentils
+- Kidney Beans
+- Basil; Bay Leaves; Rosemary; Thyme; Paprika
```

Comment:
If you mean these are all requirements, use commas instead of semicolons.

Reply:
Yeah sorry.